draft-ietf-rtcweb-jsep.xml

```diff
@@ -1976,11 +1977,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>
           <xref target="I-D.ietf-mmusic-sdp-mux-attributes" /> groups
-          SDP attributes into different categories. To avoid
-          unnecessary duplication when bundling, Section 8.1 of
-          <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation" /> specifies
-          that attributes of category IDENTICAL or TRANSPORT MUST NOT
-          be repeated in bundled m= sections. This includes m=
+          SDP attributes into different categories.
+          To avoid unnecessary duplication when bundling, attributes
+          of category IDENTICAL or TRANSPORT MUST NOT be repeated in
+          bundled m= sections, as specified in <xref
+          target="I-D.ietf-mmusic-sdp-bundle-negotiation" />, Section
+          8.1.
```

Comment:
This one looks a bit odd because you might read it as contradicting BUNDLE. I.e., it's not-repeating which is specified, not the repeating that we are now prohibiting.

Perhaps "This restates the requirements of...."